gdc_rnaseq_tools/augment_star_counts.py

```diff
@@ -0,0 +1,228 @@
+import numpy as np
+import pandas as pd
+
+from gdc_rnaseq_tools.utils import DataError, DataFormatError, get_logger
+
+
+def load_counts_table(counts):
+    '''
+    Loads per-gene counts data table as created by STAR
+
+
+    returns pandas DataFrame with the following columns:
+        gene_id
+        unstranded
+        stranded_first
+        stranded_second
+    '''
+    cdf = pd.read_table(
+        counts,
+        header=None,
+        names=['gene_id', 'unstranded', 'stranded_first', 'stranded_second'],
+    )
+    return cdf
+
+
+def load_gene_info(gene_info):
```

Comment:
These `load` functions could be a single function which takes key-word args for `read_table`:

```python
def load_table(path: str, expected_cols: list = None, **kwargs) -> pd.DataFrame:
    df = pd.read_table(path, **kwargs)
    if expected_cols:
         # do cols set test
    return df
```